File: app/src/main/java/br/com/vicentec12/mygames/MyGamesApp.java
package br.com.vicentec12.mygames;

import android.app.Application;

import br.com.vicentec12.mygames.di.AppComponent;
import br.com.vicentec12.mygames.di.DaggerAppComponent;

public class MyGamesApp extends Application {

    private AppComponent mAppComponent;

    @Override
    public void onCreate() {
        super.onCreate();
        mAppComponent = DaggerAppComponent.factory().create(this);
    }

    public AppComponent getAppComponent() {
        return mAppComponent;
    }

}

File: app/src/main/java/br/com/vicentec12/mygames/MyGamesApp.kt
package br.com.vicentec12.mygames

import android.app.Application
import br.com.vicentec12.mygames.di.AppComponent
import br.com.vicentec12.mygames.di.DaggerAppComponent

class MyGamesApp : Application() {

    lateinit var appComponent: AppComponent

    override fun onCreate() {
        super.onCreate()
        appComponent = DaggerAppComponent.factory().create(this)
    }

}